modules/phpinfo/bootstrap.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$module = new Discussion\Module(__DIR__);
+
+$module->get('/phpinfo', function() use ($module) {
```

Comment:
Potential gotcha: argument 0 to the callback is the Application from which this route was called - not the one where it was defined!